js/src/product-feed/review-request/review-request-modal.test.js

```diff
@@ -0,0 +1,100 @@
+/**
+ * External dependencies
+ */
+import { fireEvent, render } from '@testing-library/react';
+
+/**
+ * Internal dependencies
+ */
+import ReviewRequestModal from '.~/product-feed/review-request/review-request-modal';
+
+describe( 'Request Review Modal', () => {
+	it( 'Renders when is active and there are issues', () => {
+		const { queryByRole } = render(
+			<ReviewRequestModal issues={ [ '#1', '#2' ] } isActive={ true } />
+		);
+
+		expect( queryByRole( 'dialog' ) ).toBeTruthy();
+	} );
+
+	it( "Doesn't render if its not active", () => {
+		const { queryByRole } = render(
+			<ReviewRequestModal issues={ [ '#1', '#2' ] } isActive={ false } />
+		);
+
+		expect( queryByRole( 'dialog' ) ).toBeFalsy();
+	} );
+
+	it( "Doesn't render when no issues", () => {
+		const { queryByRole } = render(
+			<ReviewRequestModal issues={ [] } isActive={ true } />
+		);
+
+		expect( queryByRole( 'dialog' ) ).toBeFalsy();
+	} );
+
+	it( 'Shows maximum 5 issues and can expand the list of issues', () => {
+		const { queryByText } = render(
+			<ReviewRequestModal
+				issues={ [ '#1', '#2', '#3', '#4', '#5', '#6' ] }
+				isActive={ true }
+			/>
+		);
+
+		expect( queryByText( '#1' ) ).toBeTruthy();
+		expect( queryByText( '#2' ) ).toBeTruthy();
+		expect( queryByText( '#3' ) ).toBeTruthy();
+		expect( queryByText( '#4' ) ).toBeTruthy();
+		expect( queryByText( '#5' ) ).toBeTruthy();
+		expect( queryByText( '#6' ) ).toBeFalsy();
+
+		const button = queryByText( '+ 1 more issue(s)' );
+		expect( queryByText( 'Show less' ) ).toBeFalsy();
+		expect( button ).toBeTruthy();
+		fireEvent.click( button );
+		expect( queryByText( '#6' ) ).toBeTruthy();
+		expect( queryByText( 'Show less' ) ).toBeTruthy();
+	} );
+
+	it( 'Cancel button closes the modal', () => {
+		const onClose = jest.fn().mockName( 'onClose' );
+		const { queryByText } = render(
+			<ReviewRequestModal
+				issues={ [ '#1', '#2', '#3', '#4', '#5', '#6' ] }
+				isActive={ true }
+				onClose={ onClose }
+			/>
+		);
+
+		const button = queryByText( 'Cancel' );
+		expect( button ).toBeTruthy();
+		fireEvent.click( button );
+		expect( onClose ).toHaveBeenCalledTimes( 1 );
+	} );
+
+	it( 'Request review button calls onSendRequest after checking the checkbox', () => {
```

Comment:
💅 
The description seems to be a bit imprecise. Would it be better with something like "Request review button calls `onSendRequest` after the checkbox has been checked and clicking on the button."?